src/Form/ProductOptionForm.php

```diff
@@ -0,0 +1,442 @@
+<?php
+
+namespace Drupal\commerce_option\Form;
+
+use Drupal\commerce\EntityHelper;
+use Drupal\commerce\InlineFormManager;
+use Drupal\commerce_option\ProductOptionFieldManagerInterface;
```

Comment:
As per our Coding Standards, please group the use statements like this:

- Custom/contrib modules
- Core modules
- Php base modules

ie. 

```
use Drupal\mymodule\Entity;
use Drupal\commerce_product\Entity;

use Drupal\Core\User;

use Symfony\Component\DependencyInjection\ContainerInterface;
```